### ISSUE TITLE
Southern platforms cannot correctly connect to non-cns platforms

### ISSUE THREAD

**Issue by WenSimEHRP** [OpenTTD-China-Set/china-set-stations#1]
may be... unfixable??

**Comment by WenSimEHRP** [comment]
https://github.com/OpenTTD-China-Set/china-set-stations/blob/master/src%2Ftesting.pnml#L88-L94

```cpp
#define GET_NPLAT_FORMATION \ ((getbits(LOAD_TEMP(TILE_X_SW), NPLAT_BIT, 1) | getbits(LOAD_TEMP(TEMP_NEARBY_TILE_GRF), TILE_X_SW, 1)) << 0|\ (getbits(LOAD_TEMP(TILE_X_NE), NPLAT_BIT, 1) | getbits(LOAD_TEMP(TEMP_NEARBY_TILE_GRF), TILE_X_NE, 1)) << 1) #define GET_SPLAT_FORMATION \ ((getbits(LOAD_TEMP(TILE_X_SW), SPLAT_BIT, 1) | getbits(LOAD_TEMP(TEMP_NEARBY_TILE_GRF), TILE_X_SW, 1)) << 0|\ (getbits(LOAD_TEMP(TILE_X_NE), SPLAT_BIT, 1) | getbits(LOAD_TEMP(TEMP_NEARBY_TILE_GRF), TILE_X_NE, 1)) << 1)
```

**Comment by WenSimEHRP** [comment]
Fixed by #3
